### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by pedrorijo91**
I noticed gradle provides different issue templates at https://github.com/gradle/gradle/issues/new/choose

I'm hoping this helps reducing the amount of wrong issues opened (which has been decreasing to be fair)

image

### CODE REVIEW

**Comment by peff** [comment]
I like the idea of having templates to direct people. But I'm not sure that the download template is helping anything. Isn't the resolution there generally "nope, we're just linking to another project, and there's nothing we can do here"?

**Comment by pedrorijo91** [comment]
> I like the idea of having templates to direct people. But I'm not sure that the download template is helping anything. Isn't the resolution there generally "nope, we're just linking to another project, and there's nothing we can do here"?

@peff I think lately yes. I remember one time we were caching the page and users were receiving the wrong version. But I think we still want to know if the download pages is not working properly, as our scheduled task may have messed the DB no?

**Comment by peff** [comment]
Sure, it's definitely possible we have a broken link or whatever, and we'd want to know about that. Perhaps we should word the template to make that more clear (on the other hand, I guess people just click a link and it does or doesn't work; most of them are not going to notice that it redirected to a GitHub releases page, and then to S3).

File: .github/ISSUE_TEMPLATE/downloads.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+name: 'Donwloads on git-scm not working'
```
Comment (peff) [inline .github/ISSUE_TEMPLATE/downloads.md]:
```suggestion
name: 'Downloads on git-scm not working'
```

**Pull request merged**